manifests/charts/istio-control/istio-discovery/README.md

```diff
@@ -49,6 +49,17 @@ meshConfig:
   accessLogFile: /dev/stdout
 ```
 
+#### Configuring CNI configuration settings
```

Comment:
I hate to be someone to nitpick docs additions since docs are great... but I was wondering if this may make more sense in the CNI chart?

My logic here is that a small number of people installing this chart use CNI and may be confused , but 100% of people installing CNI use the CNI chart and the istiod chart. So we could mention there?